pkg/framework/aries/framework.go

```diff
@@ -250,19 +251,14 @@ func WithPacker(primary packer.Creator, additionalPackers ...packer.Creator) Opt
 
 // Context provides a handle to the framework context.
 func (a *Aries) Context() (*context.Provider, error) {
-	endPoint := defaultEndpoint
-	if a.inboundTransport != nil {
-		endPoint = a.inboundTransport.Endpoint()
-	}
-
 	return context.New(
 		context.WithOutboundDispatcher(a.outboundDispatcher),
 		context.WithMessengerHandler(a.messenger),
 		context.WithOutboundTransports(a.outboundTransports...),
 		context.WithProtocolServices(a.services...),
 		context.WithLegacyKMS(a.kms),
 		context.WithCrypto(a.crypto),
-		context.WithInboundTransportEndpoint(endPoint),
+		context.WithServiceEndpoint(serviceEndpoint(a)),
```

Comment:
Don't you  also need `context.WithRouterEndpoint(...)` here?

Reply:
It's not required here as only requirement for RouterEndpoint is for Route Service.